app/models/user.rb

```diff
@@ -217,6 +222,18 @@ def first_time_poster
     self.notes.where(status: 1).count == 0
   end
 
+  def follow(other_user)
```

Comment:
These don't seem to have authorization built in -- we should make it so only users can add create follows? And add tests for this.